core/src/main/scala/org/apache/spark/status/api/v1/ApplicationListResource.scala

```diff
@@ -92,7 +92,8 @@ private[spark] object ApplicationsListResource {
           sparkUser = internalAttemptInfo.sparkUser,
           completed = internalAttemptInfo.completed
         )
-      }
+      },
+      appSparkVersion = org.apache.spark.SPARK_VERSION
```

Comment:
This is not right. This might be different from the current version, just like the UI case. You have to somehow plumb it from the event log information to here.